### DISCUSSION TITLE
problem running topaz train with cryoSPARC

### DISCUSSION THREAD

**Discussion by diffracteD** [tbepler/topaz#74] [Q&A]
Hi,
I just installed topaz to use via cryosparc.
When I was trying to run Topaz-Train I'm facing the following error:
```
[CPU: 217.2 MB]  Traceback (most recent call last):
  File "cryosparc2_worker/cryosparc2_compute/run.py", line 85, in cryosparc2_compute.run.main
  File "cryosparc2_compute/jobs/topaz/run_topaz.py", line 120, in run_topaz_wrapper_train
    ver_023_plus = utils.check_topaz_version(topaz_version)
  File "cryosparc2_compute/jobs/topaz/topaz_utils.py", line 66, in check_topaz_version
    if (int(version_vals[0]) > 0) or (int(version_vals[1]) > 2) or (int(version_vals[1]) == 2 and int(version_vals[2]) >= 3):
ValueError: invalid literal for int() with base 10: ''
```

Installation looked fine and finished without any error. 
Current status of Topaz is as follow:
```
[abhisek@c101030 bin]$ topaz --version
TOPAZ 0.2.4
```

Please suggest how can I be able to fix it.
Thank you

**Comment by tbepler** [comment]
I moved this to our new discussions page, because this does not appear to be a Topaz bug and is related to integration with an external program.

Have you asked about this on the cryoSPARC forums  (https://discuss.cryosparc.com/) or looked at some of the threads there? They may have an answer to this questions already.

**Comment by alexjnoble** [comment]
Hi,

What version of CryoSparc are you using? This was a potential bug in earlier integrations of Topaz into Cryosparc, but to my knowledge has been fixed.

Best,
-Alex
